cypress/e2e/dashboard.cy.js

```diff
@@ -0,0 +1,52 @@
+describe("Dashboard", () => {
```

Comment:
Great testing. We can polish it up along with login/modal before turning it in too!